Add Intel(R) Extension for Scikit-learn samples (#617)
oneapi-src/oneAPI-samples: AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/README.md

| Time to complete                  | 25 minutes

## Purpose

Intel® Extension for Scikit-learn is a seamless way to speed up your Scikit-learn application. The acceleration is achieved through the use of the Intel(R) oneAPI Data Analytics Library ([oneDAL](https://github.com/oneapi-src/oneDAL)). Patching scikit-learn makes it a well-suited machine learning framework for dealing with real-life problems.

In this sample, you will run a SVC algorithm with Intel® Extension for Scikit-learn. You will see a significant increase in performance over the original Scikit-learn while maintaining the same precision.
  
## Key Implementation Details 
This sample code is implemented for CPU using the Python language. The example assumes you have Intel® Extension for Scikit-learn installed.

## License  
Code samples are licensed under the MIT license. See
[License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.

Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)

## Running Samples on the Intel&reg; DevCloud

> If you are running this sample on the DevCloud, see [Running Samples on the Intel&reg; DevCloud](#run-samples-on-devcloud)

Reviewer: @PetrovKP - please provide the correct url of your choice

## Build and Run the Sample

### Pre-requirement

> > NOTE: No action is required if users use Intel DevCloud as their environment.

Reviewer 2: ```suggestion
> NOTE: No action is required if you are using Intel DevCloud as your environment.
```

>   Please refer to [Intel oneAPI DevCloud](https://intelsoftwaresites.secure.force.com/devcloud/oneapi) for Intel DevCloud.

Reviewer 2: ```suggestion
  Refer to [Intel oneAPI DevCloud](https://intelsoftwaresites.secure.force.com/devcloud/oneapi) for Intel DevCloud.
```
